src/solver/mus.cpp

```diff
@@ -220,13 +231,13 @@ struct mus::imp {
         for (expr* e : mss) {
             mss_set.insert(e);
         }
-        for (expr * e : min_core) {
+        for (expr* e : min_core) {
```

Comment:
OK, fine to have * bind to the type. I don't like expr *e, which is very common in C/C++ code bases.
The reason is that I read * as part of the type expression. 
So expr and expr* are types, and 'e' is an expression of type expr*.

For that matter I also have developed a possibly peculiar convention of placing const qualifiers in post-fix mode.
I write T const & onstead of const T& so I don't have to recall what the const qualifier binds to.